src/invariant/InvariantTests.cpp

```diff
@@ -122,3 +621,82 @@ TEST_CASE("onLedgerClose fail/succeed", "[invariant]")
             app->getInvariantManager().checkOnLedgerClose(tsfp, ld));
     }
 }
+
+TEST_CASE("onBucketApply fail/succeed", "[invariant]")
```

Comment:
I would prefer to have these tests in separate file - BucketListIsConsistentWithDatabaseTests